### ISSUE TITLE
missing download links

### ISSUE THREAD

**Issue by bahajan95** [welder1976/uwow-spp#5]
hi all, pls update download links for repack, using mega NZ download as there removed the link I do not have time card download(( please can you update the link

**Comment by Crandalius** [comment]
@welder1976 

I can vouch this, DB isn't working after importing the files.

Please update the below links

Data: https://mega.nz/#!Mv41VRBY!_wn4-kvY457MDzd72dY17NRnqAF7CM2nE5MitVD6e38 DB: https://mega.nz/#!drwV2Q6A!zbUI_w1cnTfVp_FCw_IpS8YtsCIexSe9eJFSV_8zwJ8 Core: https://mega.nz/#!6YsS0QyJ!r7_4wlGQAfcEtbr25cCd-1ql8p3XIeHbUHCD7Wt2lGw

Thank you

**Comment by Crandalius** [comment]
@bahajan95  

Try this link, this is related.

ftp://spp-forum.de/Torrent/SPP-LegionV2.7z.torrent

**Comment by Crandalius** [comment]
Couldn't get it to work sadly :(

**Comment by bahajan95** [comment]
> Couldn't get it to work sadly :(

sources? or Repack?

**Comment by Crandalius** [comment]
I wasn't able to get the source work with the DB unfortunately :(

**Comment by xblubbx2** [comment]
Running the repack works (I can login with a wow client) but...

When I connect to the mysql server with user spp_user I can only see the databases legion_auth and legion_characters even though for example legion_world is present in the database/data directory.
When I try to connect as root using pw 123456 it doesn't work.
When I try to run the compiled worldserver.exe from this repo it complains that WorldDatabaseInfo isn't specified/found.

Edit: I got it to work by overriding root password with mysqld --init-file... and granting spp_user permissions to all databases. I was able to log in with the bnetserver.exe and worldserver.exe build from this repo.

I have one question though: I tested login with 2 different clients. Client 1 worked, and Client 2 gave the error that no servers/realms are available after authenticating. What is the cause of this? (Sorry, pserver noob here)
